wire-ios/Wire-iOS/Sources/UserInterface/MainController/MainTabBarController.swift

```diff
@@ -0,0 +1,94 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+func MainTabBarController(
+    contacts: UIViewController,
+    conversations: UIViewController,
+    folders: UIViewController,
+    archive: UIViewController
+) -> UITabBarController {
+
+    let mainTabBarController = UITabBarController()
+    mainTabBarController.viewControllers = [contacts, conversations, folders, archive]
+    mainTabBarController.viewControllers?[0].tabBarItem = .init(
```

Comment:
As written before an enum case would help to give the reader a better understanding of the raw integers, like `0`.